### PR TITLE
Refactor/mild style

### PR DESCRIPTION
**Pull request by LimitSpectrum**
Cleaned up some syntax and started tracking the random num generators through an enum. I think this will be easier to pass around generators rather than having to look them up and type check.

### CODE REVIEW

File: integrals.py [hunk 1]
```diff
@@ -50,16 +50,15 @@ def winding_integrand(E_i, E):
     return 2 * x_diff * p_diff / (m.pi * (x_diff ** 2 + p_diff ** 2) ** 2)
 
 
-def MC_mean_value(f, D, E_i, H_r, x_range, p_range, N, random_gen=None):
+def monte_mean_value(D, E_i, H_r, x_range, p_range, n, random_gen):
     """
-    Computes the integral of f over a rectangular domain in phase space 
-    defined by (x_range, p_range) using the mean value Monte Carlo method 
-    with N samples.
+    Computes the winding number at each point in e_i for points in a rectangular domain in
```
Comment (PrimeIdeal) [inline integrals.py]: This should be "the winding number about E_i"

Comment (LimitSpectrum) [reply integrals.py]: Something like the fix?

Comment (PrimeIdeal) [reply integrals.py]: Yup!

**Review by PrimeIdeal — APPROVED** [review]
thumbs up

**Pull request merged**